doc/reference/tooling/tt_cli/index.rst

```diff
@@ -60,9 +60,10 @@ Replacement for tarantooctl and Cartridge CLI
 ---------------------------------------------
 
 A multi-purpose tool for working with Tarantool from the command line, ``tt`` has
-come to replace :ref:`tarantoolctl <tarantoolctl>`
+come to replace the deprecated utilities :ref:`tarantoolctl <tarantoolctl>`
 and :doc:`Cartridge CLI </book/cartridge/cartridge_cli/index>` command-line utilities.
```

Comment:
Given that `Cartridge Command Line Interface` is removed from docs, shouldn't we remove the `/book/cartridge/cartridge_cli/index` link?